frontend/src/components/Navbar.tsx

```diff
@@ -0,0 +1,138 @@
+import { FC, useState, type MouseEvent } from "react";
+import { Link, NavLink } from "react-router-dom";
+
+import { useScroll } from "../lib/hooks/useScroll";
+import { capitalizeString, cn } from "../lib/utils";
+
+import { MdInsertChart, MdMenu, MdOutlineClose } from "react-icons/md";
+import { Button, buttonVariants } from "./ui/button";
+
+const NAV_LINKS = ["home", "about", "pricing", "team", "contact"] as const;
```

Comment:
I think will be better to use enum instead  of simple array. Is more type safely and have better clear descriptive names for the navigation links.

enum NavigationLinks {
  Home = "home",
  About = "about",
  Pricing = "pricing",
  Team = "team",
  Contact = "contact",
}